docs/experiments.md

```diff
@@ -8,6 +8,14 @@
 eBPF is a feature that leverages Linux kernel infrastructure to evenly distribute the load, especially in cases when there is a huge stream of messages incoming from a single appliance.
 To use the eBPF feature, you must have a host machine with and OS that supports eBPF. eBPF should be used only in cases when other ways of SC4S tuning fail. See the [instruction](./configuration.md#ebpf) for configuration details. 
 To learn more visit this [blog post.](https://www.syslog-ng.com/community/b/blog/posts/syslog-ng-4-2-extra-udp-performance)
+
+### Parallelize (TCP)
+Normally, SC4S processes incoming messages from a TCP connection in a single thread. While this works fine with many connections, it doesn't work efficiently when using a single or very few high-traffic connections. Using this feature allows SC4S to process log messages from a single high-traffic TCP connection in multiple threads, thus increasing processing performance on multi-core machines.
+
+See the [instruction](./configuration.md#parallelize)
+
+To learn more visit this [blog post.](https://www.syslog-ng.com/community/b/blog/posts/accelerating-single-tcp-connections-in-syslog-ng-parallelize)
```

Comment:
[blog post.](https://www.syslog-ng.com/community/b/blog/posts/accelerating-single-tcp-connections-in-syslog-ng-parallelize)